src/webui/yarn.lock

```diff
@@ -3671,7 +3671,7 @@ debug@^4.0.1, debug@^4.1.0, debug@^4.1.1:
   dependencies:
     ms "^2.1.1"
 
-debuglog@*, debuglog@^1.0.1:
```

Comment:
could you remove yarn.lock changes? Because PR #2866 had same changes.

Reply:
`yarn.lock` is automatically modified on build.
I wonder why it's not committed when `package.json` changed...

Reply:
emmm, the changes of `yarn.lock` should be committed by PR which changes `package.json`. but PR #2866 had done this with other dev update. I think #2866 should be merged into master. so I suggest not upload yarn.lock changes in this PR.